lib/rules/no-assignment-of-untracked-properties-used-in-tracking-contexts.js

```diff
@@ -19,15 +26,17 @@ const ERROR_MESSAGE =
  * Gets the list of string dependent keys from a computed property.
  *
  * @param {Node} node - the computed property node
+ * @param {number} dependentKeyArgumentCount - number of arguments to check for dependent keys
  * @returns {String[]} - the list of string dependent keys from this computed property
  */
-function getComputedPropertyDependentKeys(node) {
+function getComputedPropertyDependentKeys(node, dependentKeyArgumentCount = Number.MAX_VALUE) {
```

Comment:
Should these functions be extracted into `utils/computed-property-macros`?

Reply:
Good idea, I extracted them into a util file and added tests.